platforms/java/com/area9innovation/flow/Native.java

```diff
@@ -2269,7 +2291,7 @@ public static final String urlDecode(String s) {
 	@SuppressWarnings (value="unchecked")
 	public static final Object hostCall(String name, Object[] args) {
 		if (!host_call_funcs.containsKey(name)) {
-			addHostCall(name, args);
+			addHostCall(name, args.length);
```

Comment:
Why this?

Reply:
Because `addHostCall` uses only length of array of arguments, so there's no need to pass arguments as values.